config/settings.py

```diff
@@ -155,3 +155,14 @@
 
 LOGIN_URL = 'users:login'
 
+CACHE_ENABLED = True if os.getenv("CACHE_ENABLED") == "True" else False
```

Comment:
Можно указать чуть проще
```suggestion
CACHE_ENABLED = os.getenv("CACHE_ENABLED").lower() == 'true'
```